assets/app/features/upgrade/i18n/en.json

```diff
@@ -87,25 +87,26 @@
                 }
             },
             "nodes-repository-checks": {
-                "title": "Upgrading to SUSE OpenStack Cloud7 requires software repositories for all nodes",
-                "description": "Update the URLs of the following software repositories and click 'Check' to verify the repositories are available. When all reposotires have been verified click on 'Next' to proceed.",
+                "title": "Upgrading to SUSE OpenStack Cloud 7 requires updated software repositories for all nodes",
+                "description": "Prepare the software repositories locally or via SMT and update the URLs in the Administration Server configuration correspondingly. Click \"Check\" to verify all repositories are available. When all repositories have been verified click on \"Next\" to proceed.",
                 "form": {
                     "check-again": "Check"
                 }
             },
             "openstack-services": {
-                "title": "Stop OpenStack Services and Create Backup",
-                "description": "When ready, click on the button below to stop OpenStack services and create a backup of the OpenStack database.",
+                "title": "Stop all OpenStack services and back up the OpenStack database",
+                "description": "It is imperative that during and after stopping the OpenStack services no changs are made to your cloud. Once created, the backup can be found in /var/lib/crowbar/backup on Administration Server.",
```

Comment:
We should say at some point, that the *OpenStack API won't be available until the upgrade is finished*.